Src/Sarif.PatternMatcher.Sdk/ValidatorBase.cs

```diff
@@ -55,7 +55,7 @@ protected ValidatorBase()
             FingerprintToResultCache = new ConcurrentDictionary<Fingerprint, Tuple<ValidationState, ResultLevelKind, string>>();
         }
 
-        protected virtual string ScanIdentityGuid
+        public virtual string ScanIdentityGuid
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

do we need to make this change?

Reply:
No, that's a remnant from trying to get the mock tests working, reverting in next iteration.